### DISCUSSION TITLE
Format Text & Text Color Accessibility

### DISCUSSION THREAD

**Discussion by Asstrole** [windingwind/zotero-actions-tags#426] [Ideas]
I constantly use the Format Text and the Text Color drop down menus and would like some faster ways to use these. Does anyone have a script or could write one that could do any of the following:

- Make these menus (Format Text and Text Color) accessible via right click?
- Add keyboard shortcuts to the Menus (Format Text & Text Color) and/or menu items? (Text Color > Yellow Keyboard-Shortcut) ?
- Similar to "Show on Page and Unlink" popup that automatically appears when you click in quoted note text, could a similar popup be added or triggered via shortcut with the Format Text & Text Color options? 
- or any other similar helpful way to access these without so much mouse work. 

Thank you very much !

**Comment by windingwind** [comment]
It might be possible to do that with this plugin, and probably it would be better to add it to the Better Notes plugin's [Magic Key](https://github.com/windingwind/zotero-better-notes/blob/master/README.md#magic-key)

**Reply by Asstrole** [reply]
Thank you!

**Comment by Asstrole** [comment]
Should I reopen this over in Better Notes discussion?

**Reply by windingwind** [reply]
Yes, if that is a solution for you, please open an issue there.
